packages/server/src/orchestrator.ts

```diff
@@ -46,6 +46,10 @@ export function* createOrchestrator(options: OrchestratorOptions): Operation {
 
   let connectTo = `ws://localhost:${options.project.connection.port}`;
 
+  yield spawn(function* () {
+    yield createLogger({ atom: options.atom,  out: console.error });
+  });
+
```

Comment:
Since `createLogger({ atom: options.atom,  out: console.error })` is an operation, you can spawn it directly. No need to wrap it in an operation that just yields to it:

```suggestion
  yield spawn(createLogger({ atom: options.atom,  out: console.error }));
```